config/clients/java/template/publish.gradle.mustache

```diff
@@ -51,10 +51,10 @@ signing {
 nexusPublishing {
     repositories {
         sonatype {
-            nexusUrl.set(uri('https://s01.oss.sonatype.org/service/local/'))
-            snapshotRepositoryUrl.set(uri('https://s01.oss.sonatype.org/content/repositories/snapshots/'))
-            username.set(System.getenv('MAVEN_USERNAME'))
-            password.set(System.getenv('MAVEN_PASSWORD'))
+            nexusUrl.set(uri('https://central.sonatype.com/'))
```

Comment:
I am not sure these are the right URLs, they may need to be:
* nexusUrl.set(uri("https://central.sonatype.com"))
* snapshotRepositoryUrl.set(uri("https://central.sonatype.com/repository/maven-snapshots/"))

But let's test - let's pair up with @jimmyjames on Friday and go through the process with our dummy test artifact: https://central.sonatype.com/artifact/dev.openfga/java